packages/client/modules/meeting/components/MeetingCheckInPrompt/NewCheckInQuestion.tsx

```diff
@@ -117,6 +128,7 @@ const NewCheckInQuestion = (props: Props) => {
         readOnly={!isFacilitating}
         placeholder='e.g. How are you?'
         editorRef={editorRef}
+        updateAndroidQuestion={updateQuestionAndroidFallback}
```

Comment:
-1 to make it consistent I'd name this prop something like `setEditorStateFallback`, this will make it clear it'll only be called when normal editor isn't working